geode-core/src/main/java/org/apache/geode/internal/cache/locks/TXOriginatorRecoveryProcessor.java

```diff
@@ -57,9 +56,6 @@ static void sendMessage(Set members, InternalDistributedMember originator, TXLoc
     msg.processorId = processor.getProcessorId();
     msg.txLockId = txLockId;
 
-    // send msg to all members EXCEPT this member...
-    Set recipients = new HashSet(members);
-    recipients.remove(dm.getId());
     msg.setRecipients(members);
```

Comment:
@pivotal-eshu does this change look right?  The comment seems misleading.

Reply:
`recipients` was the unused container, so this logic has been wrong for awhile (if indeed that was the intended behavior).  If the logic is wrong, then we should make a jira story to correct it and reference the LGTM issue there

Reply:
It looks like #4586 corrects this

Reply:
Yup, that is corrected in that after discussion with Gester